Felienne/hedy
# [Fix] Turtle not showing when used in combination with ask keyword

Provide a general summary of your changes in the Title of the PR

**Description**
Currently when using the ask keyword in combination with the turtle drawing (in any possible format) the arrow of the turtle won't show. This occurred due to the turtle canvas being empties when a user input is read, but with this operation the arrow is removed as well due to being initialized before the input function is called. By replacing `empty()` with `hide()` and write an additional check to verify if we have a turtle combined with `show()` the error should be solved.

**Fix for**
This PR fixed #1361.

**How to test**
Run the code as described in the issue and verify that the arrow is showing in this case. Code can be found below as well:

```
ask Wat is je lievelingskleur?
turn right
forward 50
turn left
forward 50
```

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
